system/objects.rst

```diff
@@ -88,15 +88,97 @@ to Zammad. If you made a mistake or just want to discard your changes, click
 System attributes
 -----------------
 
-Zammad comes with pre-configured attributes. Some of these currently do not
-provide the possibility to edit them via UI (or at all).
+Zammad comes with pre-configured attributes. Some of them can't be edited via
+UI (or at all).
 
 This is not a bug but is to save you from possibly nuking Zammad.
 
    .. tip::
 
       There are technical exceptions which can be solved via console.
-      This e.g. affects ticket states and priorities, see
-      :docs:`console section </admin/console.html>`.
+      See :docs:`console section </admin/console.html>` for further information.
 
       💰 If you're a hosted customer, please contact your support for more. 💰
```

Comment:
Is this still relevant? Perhaps we should clarify this. We actually did the new feature to reduce support requests.

Reply:
I removed that part. I think none of the other ticket attributes should (and can?) be manipulated via console.